src/foam/nanos/auth/UserRegistrationDAO.js

```diff
@@ -65,6 +67,15 @@ and then
           Loggers.logger(x, this).error("Theme not found");
           throw new RuntimeException("Theme not found for user registration");
         }
+        Group group = (Group) ((DAO) getX().get("groupDAO")).find_(getX(), user.getGroup());
```

Comment:
Shouldnt this be testing the registrationGroup from the theme? That way we can test the requirements of the group the user will be assigned rather than having one group's requirements depend on the configuration of another